apps/client/src/pages/WrapperPage.js

```diff
@@ -35,36 +67,74 @@ const WrapperPage = ({ checker, topic, resetChecker }) => {
             highlightActive
           >
             <StepByStepItem
-              active={!finishedLocation}
-              checked={finishedLocation}
+              active={
+                (isActive("location") ||
+                  isActive("address") ||
+                  !activeComponents) &&
+                !isFinished("address")
```

Comment:
Holy moly, this is a huge condition. Can we not just use `!isFinished("address")`?